CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Sambacry exploiter. #1567
 - "Kill file" option in the config. #1536
 - Netstat collector, because network connection information wasn't used anywhere. #1535
+- Checkbox that allowed the dropper to first try move itself. #1537
```

Comment:
Instead of "allowed", perhaps something like "gave the option to not", since we're changing it to always move now.